doc/en/user/source/installation/upgrade.rst

```diff
@@ -49,6 +49,9 @@ classes/plugins generating HTML output may need to update their settings. For mo
     It is recommended that static web files be disabled if they are not necessary in order to
     mitigate cross-site scripting attacks. For more information, see :ref`tutorials_staticfiles`.
 
+However CSP problems can also be introduced in your environment when running GeoServer behind a proxy server.
```

Comment:
Don't constrain it to proxy, it can be due to some leftover CSP issues in extension/community modules or less-used pages. Mind, a CSP issue due to a plugin component in a page might brick the entire page.

Reply:
Okay I came up with much better wording, please review.